checkpoint/checkpoint/app.py

```diff
@@ -61,8 +68,8 @@
     app.logger.setLevel(gunicorn_logger.level)
     logger.setLevel(gunicorn_logger.level)
 
-with open("/etc/hostname", "r") as f:
-    analytics = AnalyticsClient(f.readline().strip(), app.logger)
+hostname = socket.gethostname()
```

Comment:
@JoshBroomberg Please review. `/etc/hostname` does not exist on Mac.

Reply:
I remember thinking "this will be fine, it only runs in a container" but that clearly isn't true for local dev so... yep